Add Charter (#592)
nodejs/package-maintenance: Governance.md

## Package Maintenance Working Group Charter

The Node.js Package Maintenance Working Group is jointly governed by the members
of that Working Group (WG), and maintains oversight of its constituting teams:
the `@nodejs/package-maintenance` and `@nodejs/corepack` teams.

The WG responsibilities include:

> * Management of the "version management tools" shipped with the Node.js distribution (see below).

Reviewer: I’m not sure whether whatever potential new tool we make would necessarily ship bundled in the distribution; and of course we might unbundle Corepack. We should still be responsible for these things regardless, though, so I think a better way to put it is that we’re responsible for anything under https://github.com/nodejs.

```suggestion
* Management of the “version management tools” maintained within the Node.js GitHub org.
```

Reviewer 2: Ah interesting point. Hm, yeah I think this is a good change but npm is not maintained within this org, so that feels at odds right?

* Management of repositories within the [pkgjs](https://github.com/pkgjs)
  GitHub organization including but not limited to:
    * Managing the list of organization owners which supplement the standard
      Node.js organization owners as outlined in:
[https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners](https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners)
    * Overseeing new repositories (creating, moving, removing).
    * Managing the maintainer teams for all of the repositories.
    * Contribution policy for repositories.
* Technical direction for the projects within
  the [pkgjs](https://github.com/pkgjs) organization.
* Project governance and process (including this policy).
* Managing the maintainer teams and contribution policies for the
  following repositories:
  * nodejs/ci-config-travis
  * nodejs/ci-config-github-actions
  * nodejs/package-maintenance

For the current list of WG members, see the project [README.md][].

[README.md]: ./README.md#current-project-team-members

> ### Version Management Tools

Reviewer 2: For folks seeing such a large diff and wondering what changed, this is the section to focus on. The rest is marked as changed but really isnt. Additionally, the parts above about the repo's is because the goal here is to subsume the corepack team into the PMWG.

The WG has authority over Version Management solutions in the project including:

> * Maintaining the relationship[^note] with any tools included in the Node.js default distribution with the express purpose of the following:

Reviewer: ```suggestion
* Maintaining the relationship[^note] with any tools included or referenced by Node.js-managed software with the express purpose of the following:
```

  - Download or install additional software from sources outside the default distribution.
  - Download or install additional Node.js distributions (default or third-party).

> * Contribution policy for all of the additional software included in the Node.js distribution as qualified above.

Reviewer: ```suggestion
* Contribution policy for all of the additional software within the Node.js GitHub organization as qualified above.
```

Reviewer 2: Hm interesting change. I think I understand why but would you mind explaining the goal of this wording over the other?

* Conduct guidelines for the Version Management group.
* Recommend technical solutions for Version Management for the Node.js ecosystem.

[^note]A note on the meaning of "maintaining the relationship":

> This means proposals to change the relationship between Node.js and npm or other tools (from either side) should be presented to the PMWG first to work out the details of the proposed changed.

Reviewer: ```suggestion
This means proposals to change the relationship between Node.js and npm or other tools (from either side) should be presented to the PMWG first to work out the details of the proposed change.
```
